script/Arcueid/Day 9/09_00_ARC09_4_2.txt

```diff
@@ -106,17 +118,18 @@ I mutter to myself before sitting up.
 [sha:5684dbfa7323cf3f063516748b83823182ce62ae]{
 -- Page 243, Offset 21969.
 -- 明かりがついていないのに、視界が白黒に点滅した。
-Even though the lights aren't on, my vision flickers between black and white.
+// Bright flash vfx that's really more white than black.
+The lights aren't on, but my vision still flashes white.
```

Comment:
Yeah, I originally had this along the lines of "colour fades from my vision", but the game vfx here are more of a strobe flash, so I re-cut it to that. It also makes a bit more sense combined with the 'lights aren't on' - lights aren't on but you see light -> sensible, lights aren't on but your vision goes black and white -> ??? ok. Not wedded to the change though.

Reply:
I mean regardless of what _makes sense_, I don't really think the interpretation that his vision is flashed with light fits the text.

Reply:
Well, I think it can be a good flag - if it doesn't make sense, maybe we're missing something. I'll check with Petrikow that this _is_ just weird writing, and if so will go back to that interpretation.